### PR TITLE
New shortcode for embedding VIDEO MP4.

### PR DESCRIPTION
**Pull request by micronAMZN**
## Change summary

Commit includes shortcode, test video and poster image. Style guide media page updated. Smoketest updated. Removed a bunch of odd references to another project from smoketest.

Signed-off-by: Mike Cronin <58789750+micronAMZN@users.noreply.github.com>

### Submission Checklist:

* [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

@netlify /docs//contributing/to-docs/style-guide/media/

### CODE REVIEW

**Comment by amzn-leenguy** [comment]
I approve!

**Review by amzn-leenguy — APPROVED** [review]

File: content/smoketest.md [hunk 1]
```diff
@@ -232,26 +223,35 @@ character. The square brackets contain the image's alt text. Try to always use
...
 
-Even if you choose not to use the figure shortcode, an image can also be a link. This
-time the pencil icon links to the Kubernetes website. Outer square brackets enclose
+```markdown
+{{</* image-width "/images/welcome-guide/ui-editor-labeled.png" "700" "An annotated image of O3DE editor's user interface." */>}}
+```
+
+`image-width` example output:
+
+{{< image-width "/images/welcome-guide/ui-editor-labeled.png" "700" "An annotated image of O3DE editor's user interface." >}}
```
Comment (willihay) [inline content/smoketest.md:245]: This example reminds me that I think we should have named parameters in this shortcode.

File: content/docs/contributing/to-docs/style-guide/media.md [hunk 2]
```diff
@@ -123,13 +123,76 @@ Artful images are used for artistic or marketing purposes, such as images that d
...
+1. `poster="/images/<image.png>"` - A static image that displays while the video loads or if the video fails to load. This image should be the same size and aspect ratio as the video. 
+1. `autoplay="true"` - The video plays as soon as it loads.
+1. `loop="true"` - The video plays in a loop.
+1. `width="<200>"` - Scale the video by specifying a width in pixels.
+1. `muted="true"` - Video is muted if an audio track exists.
+1. `type="video/<mp4 OR ogg OR webm>"` - Video type. MP4 is default.
+
+The are two additional options that are always enabled.
+
+1. `preload=auto` - Loads the video with the page.
```
Comment (willihay) [inline content/docs/contributing/to-docs/style-guide/media.md:170]: nit, double quotes for consistency:
```suggestion
1. `preload="auto"` - Loads the video with the page.
```

File: content/smoketest.md [hunk 3]
```diff
@@ -447,27 +447,64 @@ You can add inline O3DE GUI icons with the `icon` shortcode. Icon `.svg` files a
...
+1. `poster="/images/<image.png>"` - A static image that displays while the video loads or if the video fails to load. This image should be the same size and aspect ratio as the video. 
+1. `autoplay="true"` - The video plays as soon as it loads.
+1. `loop="true"` - The video plays in a loop.
+1. `width="<200>"` - Scale the video by specifying a width in pixels.
+1. `muted="true"` - Video is muted if an audio track exists.
+1. `type="video/<mp4 OR ogg OR webm>"` - Video type. MP4 is default.
+
+The are two additional options that are always enabled.
+
+1. `preload=auto` - Loads the video with the page.
```
Comment (willihay) [inline content/smoketest.md:483]: nit, double quotes for consistency:
```suggestion
1. `preload="auto"` - Loads the video with the page.
```

**Review by willihay — APPROVED** [review]

**Review by chanmosq — APPROVED** [review]
Tested on Safari, all is good!

**Pull request merged**